pkg/kubelet/metrics/collectors/resource_metrics.go

```diff
@@ -41,6 +41,13 @@ var (
 		metrics.ALPHA,
 		"")
 
+	nodeSwapUsageDesc = metrics.NewDesc("node_swap_usage_bytes",
```

Comment:
do we need "available" as well. Total allocated swap size

Reply:
This metric is covered here: https://github.com/google/cadvisor/pull/3339.
I've added a `machine_swap_bytes` metrics for that purpose.